### PR TITLE
Configure cookies for HttpOnly and secure cookies

### PR DESCRIPTION
**Pull request by gracez72**
[TASK-33930](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_sprints/taskboard/Adaptive%20Cards/Intern%20GitHub/Y20-VW/09?workitem=33930)

This PR changes server to require secure cookies and configures site cookies for HttpOnly for compliance requirements. It also enables strict HSTS headers to pass SSLScanner tests.

**Main file to review:** `server/src/app.ts`

### CODE REVIEW

File: private-templates-service/server/src/app.ts [hunk 1]
```diff
@@ -20,6 +22,19 @@ app.use(passport.initialize());
 app.use(passport.session());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
+app.use(helmet.hsts({
+  maxAge: 15552000,
+  includeSubDomains: false
```
Comment (beervoley) [inline private-templates-service/server/src/app.ts]: Don't disable subDomains. We do not use any for now, but if somebody will want to host some part of the website on subdomain then hsts won't be enabled there :)

Comment (beervoley) [reply private-templates-service/server/src/app.ts]: Also, please remove **package.json** from the PR. 
run **git checkout origin/dev -- package.json** in the correct directory

File: private-templates-service/server/src/app.ts [hunk 2]
```diff
@@ -20,6 +22,18 @@ app.use(passport.initialize());
 app.use(passport.session());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
+app.use(helmet.hsts({
+  maxAge: 15552000
```
Comment (kodyang) [inline private-templates-service/server/src/app.ts]: whats this

File: private-templates-service/server/src/app.ts [hunk 3]
```diff
@@ -20,6 +22,18 @@ app.use(passport.initialize());
...
+}));
+app.use(session({
+  secret: '#{CLIENT_ID_TOKEN}#',
+  cookie: {
```
Comment (kodyang) [inline private-templates-service/server/src/app.ts]: is this a cookie we need compliance for

File: private-templates-service/server/src/app.ts (hunk 2 above)
Comment (kodyang) [reply private-templates-service/server/src/app.ts]: oh 180 days

File: private-templates-service/server/src/app.ts (hunk 3 above)
Comment (gracez72) [reply private-templates-service/server/src/app.ts]: Yeah this is for if we do use cookies to only use secure ones

**Pull request merged**